[2024-04-11T14:26:26.506+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-11T14:26:26.517+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_gg_bigquery_v02.create_retail_dataset scheduled__2024-04-09T17:00:00+00:00 [queued]>
[2024-04-11T14:26:26.523+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_gg_bigquery_v02.create_retail_dataset scheduled__2024-04-09T17:00:00+00:00 [queued]>
[2024-04-11T14:26:26.523+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 3
[2024-04-11T14:26:26.534+0000] {taskinstance.py:2327} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_retail_dataset> on 2024-04-09 17:00:00+00:00
[2024-04-11T14:26:26.543+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2083) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-11T14:26:26.542+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_gg_bigquery_v02', 'create_retail_dataset', 'scheduled__2024-04-09T17:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_gg_bigquery.py', '--cfg-path', '/tmp/tmpw7nhahn0']
[2024-04-11T14:26:26.544+0000] {standard_task_runner.py:91} INFO - Job 13: Subtask create_retail_dataset
[2024-04-11T14:26:26.544+0000] {standard_task_runner.py:63} INFO - Started process 2085 to run task
[2024-04-11T14:26:26.573+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_gg_bigquery_v02.create_retail_dataset scheduled__2024-04-09T17:00:00+00:00 [running]> on host a4f2bd86ee13
[2024-04-11T14:26:26.621+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='duy' AIRFLOW_CTX_DAG_ID='dag_with_gg_bigquery_v02' AIRFLOW_CTX_TASK_ID='create_retail_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-04-09T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-09T17:00:00+00:00'
[2024-04-11T14:26:26.623+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-11T14:26:26.628+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-04-11T14:26:26.628+0000] {base.py:84} INFO - Using connection ID 'gcp' for task execution.
[2024-04-11T14:26:26.689+0000] {bigquery.py:470} INFO - datasetId was not specified in `dataset_reference`. Will use default value retail.
[2024-04-11T14:26:26.689+0000] {bigquery.py:470} INFO - projectId was not specified in `dataset_reference`. Will use default value ***-projecct.
[2024-04-11T14:26:26.690+0000] {bigquery.py:481} INFO - Creating dataset: retail in project: ***-projecct 
[2024-04-11T14:26:28.138+0000] {bigquery.py:485} INFO - Dataset created successfully.
[2024-04-11T14:26:28.173+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-11T14:26:28.189+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_gg_bigquery_v02, task_id=create_retail_dataset, execution_date=20240409T170000, start_date=20240411T142626, end_date=20240411T142628
[2024-04-11T14:26:28.249+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-11T14:26:28.268+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-11T14:26:28.269+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
